images/cluster-api/Dockerfile

```diff
@@ -13,13 +13,40 @@
 # limitations under the License.
 
 # Includes go, gcloud, and etcd
-FROM gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
```

Comment:
Any reason we don't just bump the go version in https://github.com/kubernetes/test-infra/blob/master/images/gcloud/Dockerfile and then rebase this image off the new tag? @BenTheElder ?

Reply:
that image is sorta deprecated, sorry I missed this comment. we don't use it for much, kubekins-e2e despite the name is sort of a common kubernetes test runtime versioned by k8s release, we maintain that because of this. otherwise we'd generally prefer new jobs use https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md and use standard docker images rather than some special image from us.